docs/UserGuide.md

```diff
@@ -83,18 +83,39 @@ A person can have any number of tags (including 0)
 </div>
 
 Examples:
-* `add n/John Doe e/johnd@example.com c/Hong Kong`
-* `add n/John Doe e/asdf@gmail.com c/Singapore p/61234567 t/Internal`
+* `add n/John Doe e/johnd@example.com c/HK`
+* `add n/John Doe e/asdf@gmail.com c/SG p/61234567 t/Internal`
 
-### Listing all persons : `list`
+### Adding a job: `add_job`
 
-Shows a list of all persons in the address book.
+Adds a job to the job list.
 
-Format: `list`
+Format: `add_job ti/TITLE [d/DESCRIPTION] v/VACANCY`
+
+Examples:
+* `add_job ti/Software Engineer d/Must be proficient in C++ v/10`
+* `add_job ti/ML Engineer v/15`
+
+### Adding an application: `add_app`
+
+Adds a job application from a candidate to the application list.
+
+Format: `add_app e/EMAIL ti/TITLE [s/STATUS]`
+
+* The application added will contain the candidate with the specified email and the job with the specified title.
+* If `STATUS` is omitted, then the `STATUS` of the application will be `PRESCREEN`.
+* There cannot be 2 or more applications with the same pair of candidate and job.
```

Comment:
I don't mind being more specific, though should we only be this specific if candidate and job are in the same sentence? Since we are using just "email" and "title" just about everywhere.